default-fields/0000-default-fields.md

```diff
@@ -28,9 +233,29 @@ Why should we *not* do this?
 # Alternatives
 [alternatives]: #alternatives
 
-What other designs have been considered? What is the impact of not doing this?
+Just stick with builders and constructor functions.
```

Comment:
Yeh, this is basically a placeholder. It turns out writing about alternatives and drawbacks is kind of difficult.